setup.py

```diff
@@ -105,7 +105,7 @@ def get_git_sha():
         "slackclient>=2.6.2",
         "sqlalchemy>=1.3.16, <2.0",
         "sqlalchemy-utils>=0.36.6,<0.37",
-        "sqlparse>=0.3.0, <0.4",
+        "sqlparse==0.3.0",
```

Comment:
Can we add a comment right here as to why it's pinned? Maybe a link to an issue on the lib's Github (if any) or to this PR if that's the best option
```
"sqlparse==0.3.0",  # PINNED! see PR #10165
```

Reply:
done

Reply:
done